tests/test_generic/tests/test_xml.py

```diff
@@ -27,10 +27,17 @@ def test_xml_files(self):
     def _check_files_in_path(self, module):
         path = get_industry_path() + module
         is_studio_required = False
+        static_files = set()
+        in_use_files = set()
         for root, dirs, files in os.walk(path):
+            dirs.sort(
+                reverse=True
+            )  # sort the directory by alphabetical order so static directory is read first.
             for file_name in files:
                 file_path = os.path.join(root, file_name)
                 ext = os.path.splitext(file_path)[1].lower()
+                if 'static/src/binary' in file_path:
```

Comment:
I don't know exactly how to proceed, but there are more paths to check. See for instance photography, with static/demo/img ...

Reply:
Taken into account